<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrumpDialog</class>
 <widget class="QDialog" name="TrumpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trump Dialog</string>
  </property>
  <widget class="QLabel" name="trumpLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trump</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectCardButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select card...</string>
   </property>
  </widget>
  <widget class="QLabel" name="partnerCardLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Partner Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectTrumpButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select trump...</string>
   </property>
  </widget>
  <widget class="QLabel" name="trumpSuitLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>No Trump Selected</string>
   </property>
  </widget>
  <widget class="QLabel" name="trumpSuitLabel_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>No Card Selected</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
